README.md

```diff
@@ -417,220 +592,31 @@ Add into the server configuration file's authorization section.
-  "out_bytes": 55162353,
-  "slow_consumers": 0,
-  "http_req_stats": {
-    "/": 223,
-    "/connz": 31,
-    "/routez": 1,
-    "/subsz": 7,
-    "/varz": 35
-  }
-}
-```
-
-### /connz
-
-<a href="http://localhost:8222/connz" target="_blank">http://localhost:8222/connz</a> reports more detailed information on current connections. It uses a paging mechanism which defaults to 1024 connections.
-You can control these via url arguments (limit and offset), e.g. <a href="http://localhost:8222/connz?limit=1&offset=1" target="_blank">http://localhost:8222/connz?limit=1&offset=1</a>.
-You can also report detailed subscription information on a per connection basis using subs=1, e.g. <a href="http://localhost:8222/connz?limit=1&offset=1&subs=1" target="_blank">http://localhost:8222/connz?limit=1&offset=1&subs=1</a>.
-
-```json
-{
-  "now": "2016-04-26T16:40:59.926732854-04:00",
-  "num_connections": 2,
-  "offset": 0,
-  "limit": 1024,
-  "connections": [
-    {
-      "cid": 1,
-      "ip": "73.170.105.42",
-      "port": 46422,
-      "start": "2016-04-25T22:09:34.715492478-04:00",
-      "last_activity": "2016-04-26T16:40:56.17467432-04:00",
-      "uptime": "18h31m25s",
-      "idle": "3s",
-      "pending_bytes": 0,
-      "in_msgs": 1112,
-      "out_msgs": 0,
-      "in_bytes": 139101,
-      "out_bytes": 0,
-      "subscriptions": 1,
-      "lang": "go",
-      "version": "1.1.9"
-    },
-    {
-      "cid": 3,
-      "ip": "129.192.191.2",
-      "port": 55072,
-      "start": "2016-04-25T22:09:36.685630786-04:00",
-      "last_activity": "2016-04-26T16:40:56.17467432-04:00",
-      "uptime": "18h31m23s",
-      "idle": "3s",
-      "pending_bytes": 0,
-      "in_msgs": 0,
-      "out_msgs": 1112,
-      "in_bytes": 0,
-      "out_bytes": 139101,
-      "subscriptions": 1,
-      "lang": "go",
-      "version": "1.1.9"
-    }
-  ]
-}
-```
-
-### /routez
-
-<a href="http://localhost:8222/routez" target="_blank">http://localhost:8222/routez</a> reports information on active routes for a cluster. Routes are expected to be low, so there is no paging mechanism currently with this endpoint. It does support the subs arg line /connz, e.g. <a href="http://localhost:8222/routez?subs=1" target="_blank">http://localhost:8222/routez?subs=1</a>
-
-```json
-{
-  "now": "2015-07-14T13:30:59.349179963-07:00",
-  "num_routes": 1,
-  "routes": [
-    {
-      "rid": 1,
-      "remote_id": "de475c0041418afc799bccf0fdd61b47",
-      "did_solicit": true,
-      "ip": "127.0.0.1",
-      "port": 61791,
-      "pending_size": 0,
-      "in_msgs": 0,
-      "out_msgs": 0,
-      "in_bytes": 0,
-      "out_bytes": 0,
-      "subscriptions": 0
-    }
-  ]
-}
-```
-
-### /subsz
+If the monitoring port is enabled, the NATS server runs a lightweight HTTP server that has the following endpoints: /varz, /connz, /routez, and /subsz. All endpoints return a JSON object. See [NATS Server monitoring](http://nats.io/documentation/server/gnatsd-monitoring/) for endpoint examples.
```

Comment:
There is `/stacksz` too in case should be added (does not return a JSON object though).